src/components/NavigationTabs/index.tsx

```diff
@@ -118,7 +119,13 @@ export function AddRemoveTabs({
           <StyledArrowLeft stroke={theme.text2} />
         </HistoryLink>
         <TYPE.mediumHeader fontWeight={500} fontSize={20}>
-          {creating ? 'Create a pair' : adding ? 'Add Liquidity' : 'Remove Liquidity'}
+          {creating ? (
+            <Trans id="pools.buttons.createPair">Create a pair</Trans>
```

Comment:
Capitalization seems to be inconsistent

Reply:
I think I'll leave this one as-is fo rnow